index.html

```diff
@@ -38,7 +38,7 @@ <h2>Haiku</h2>
                 “Two things are infinite: the universe and human stupidity; and I'm not sure about the universe.”<br>
                 &mdash; <em>Albert Einstein</em>
             </blockquote>
-            <p>&copy; 2019 <a href="mailto:justin.wagner@student.saultcollege.ca?subject:Message from CSD120 Website">Justin Wagner</a>
+            <p>&copy; 2019 <a href="mailto:justin.wagner@student.saultcollege.ca?subject=Message%20from%20CSD120%20Website">Justin Wagner</a>
```

Comment:
Use `=` not `:` to set the subject.  Use `%20` to URL-encode the space character